internal/pkg/flags/flags.go

```diff
@@ -52,6 +52,8 @@ const (
 	DefaultWatcherResourceLimitsMemory                                  = "200Mi"
 	DefaultDropStoredVersion                                            = "v1alpha1"
 	DefaultMetricsCleanupIntervalInMinutes                              = 15
+	DefaultKlmControllerManagerName                                     = "klm-controller-manager"
+	DefaultKlmControllerManagerNamespace                                = "kcp-system"
```

Comment:
we already have `DefaultIstioGatewayNamespace` use `kcp-system`, basically istio gateway and klm are deployed under same namespace, think about reuse existing flag, also consider renaming `DefaultIstioGatewayNamespace` since this namespace is not only for istio gateway anymore.